ably/rest_client.go

```diff
@@ -35,6 +36,23 @@ func query(fn func(context.Context, string, interface{}) (*http.Response, error)
 	}
 }
 
+type Raw []byte
```

Comment:
As an exported type with exported methods, please can we add a comment line above the type to explain what this represents. Also a comment line above each exported method to explain their use. Main reason for adding this is to improve the auto generated docs at [pkg.go.dev](https://pkg.go.dev/github.com/ably/ably-go/ably)

Reply:
Actually I don't think this is meant to be exported whoops. Thanks.